locales/en/messages.json

```diff
@@ -574,13 +586,15 @@
         "message": "We also have a <a href=\"https://www.facebook.com/groups/betaflightgroup/\" target=\"_blank\">Facebook Group</a>.<br />Join us to get a place to talk about Betaflight, ask configuration questions, or just hang out with fellow pilots."
     },
     "defaultChangelogAction": {
-        "message": "Changelog"
+        "message": "Changelog",
+        "description": "Please translate tabChangelog instead"
```

Comment:
Why is this needed? Do you want that the translators let this as is and go to the other translation and modify the content (that is a reference) with the real translation? This will not happen for sure...

Reply:
I may be making assumptions based on a misunderstanding.

This PR adds a string tabChangelog, which will show as a new string requiring translation

I can't predict if we're too close to release to expect that it will be translated, but since its the same content as defaultChangelogAction, we can use that.

At some point we remove defaultChangelogAction from `fallback` and `en`, and I think the removal from `en` will remove it from the other languages automatically.

Reply:
The translations works in this way:
- If we change/add a translation, the translators will be notified to do the job. If some of them don't do it, it must fallback to English.
- We let two or three days before releasing to let the translators finish the language. If some of them don't do it, it will be English.
- Adding the fallback messages file, what we have is what I asked in the other PR: a new file where we can put the constants messages (it will not be translated). I don't know if this is what we want.

Don't worry if we lose a translation (they are two little strings). We must do code easy to maintain and if we make this hacks, they will last forever.

Reply:
calling it constants kind of bothered me. something in the direction of `locales/not_translated/messages.json` would be a bit more clear. 

For these hacks we could commit to a time to rip them out.

Reply:
I agree with @McGiverGim's point - we are not in a situation where we can expect all of our translations to be 100% complete - after all they are done by volunteers, and we can't delay releases indefinitely in order to give them time to complete translations.
In this environment, it is probably better to not introduce more complexity in order to enable workarounds for translation keys that are changed - simply introduce the new key, and wait for the translators to do their work in the time they need, just like we are already doing for new translation keys that are added.